x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -177,23 +177,28 @@ const IntegrationPostureDashboard = ({
 export const ComplianceDashboard = () => {
   const [selectedTab, setSelectedTab] = useState(CSPM_POLICY_TEMPLATE);
   const getSetupStatus = useCspSetupStatusApi();
-  const hasFindings = getSetupStatus.data?.status === 'indexed';
+  const hasFindingsKspm =
+    getSetupStatus.data?.kspm?.status === 'indexed' ||
+    getSetupStatus.data?.indicesDetails[0].status === 'not-empty';
```

Comment:
Is it guaranteed always to be the first index of the array?

Reply:
<img width="403" alt="Screen Shot 2023-03-22 at 3 39 23 PM" src="https://user-images.githubusercontent.com/8703149/227054232-f3fa1a9e-5cce-4d60-bb62-2360dc7ad704.png">
yep